SportsHub/SportsHub.Core/Models/Article.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace SportsHub.Domain.Models
+{
+    public class Article
+    {
+        public Article()
+        {
+            Comments = new HashSet<Comment>();
```

Comment:
I don`t think that we need add empty HashSet here

Reply:
Okay, I've removed all `HashSets` from the constructors but everyone should remember to initialize the collections before trying to use them.